Documentation/ApiOverview/BackendModules/CliScripts/Index.rst

```diff
@@ -54,8 +54,9 @@ A simple example can be found in the :php:`ListSysLogCommand`:
     /**
      * Configure the command by defining the name, options and arguments
      */
-    public function configure()
+    protected function configure()
     {
+        parent::configure();
```

Comment:
Well, I disagree. Only because PHP devs are super lazy regarding parent calls there is no real reason not to do them. In this special case the parent method is empty. But will it always be?
It's better to have proper code in place right upfront than adding such stuff later, because something broke. This is solely for forward compatibility.

Reply:
Here's what's on my mind about this:

1. The empty `configure()` exists since [its inception](https://github.com/symfony/symfony/commit/6feaecf2fa29707703c3068fe915113597a3cfc6). So there was not a single change here within 9 years.
2. AFAIS the method only exists to ensure it is callable from the constructor without any intention to really put logic into it.
3. As said, the docs don't mention this at all, so it cannot be important.
4. _If_ we go this way, TYPO3 should be a role model and do this first.

Reply:
Any update? Could you two align on some result / solution here?

Reply:
1. Symfony docs is not perfect
2. One should always call the parent for overwritten function unless there is a good reason not to do so and an according documentation.
3. They failed to make the parent class `abstract`? They failed to document that this method will never be filled with functionality and therefore does not need to be called?

Overall this is just bad code design.
But I don't have a very strong opinion on how our docs should look like.
Feel free to change that as you consider this correct.

Reply:
I'm not adamant on this, but I think my point 4. holds true. Before we change the docs like this, TYPO3 should do this in its code.